DataFormats/HeavyIonEvent/src/classes_def.xml

```diff
@@ -16,6 +16,10 @@
  <version ClassVersion="10" checksum="934677092"/>
 </class>
 <class name="edm::Wrapper<pat::HeavyIon >" />
+<class name="pat::HFFilterStruct" ClassVersion="3">
+ <version ClassVersion="3" checksum="2657003700"/>
```

Comment:
why not ClassVersion=1?

Reply:
Looking into this twiki: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts

I found the following statement: "Version numbers 0, 1, and 2 have special meaning to ROOT and should not be used as version numbers. By convention, CMS will use 3 as the initial version number of a class which has never been stored before"

Based on that I choose ClassVersion=3. However I tested ClassVersion=2 and ClassVersion=1. For ClassVersion=2, it compiles with no problem, for ClassVersion=1, I got  this error message:

$ error: for class 'pat::HFFilterStruct' ROOT says the ClassVersion is -1 but classes_def.xml says it is 1. Are you sure everything compiled correctly?

Reply:
in the same twiki "By convention, CMS will use 3 as the initial version number of a class which has never been stored before. "

Reply:
Ah, good point, I did not know that. Thanks for checking!